src/installer/tests/HostActivation.Tests/DependencyResolution/AdditionalProbingPath.cs

```diff
@@ -114,12 +114,12 @@ public SharedTestState()
                     .AddMicrosoftNETCoreAppFrameworkMockCoreClr(TestContext.MicrosoftNETCoreAppVersion)
                     .Build();
 
-                string nativeDependencyRelPath = $"{TestContext.TargetRID}/{Binaries.GetSharedLibraryFileNameForCurrentPlatform("native")}";
+                string nativeDependencyRelPath = $"{TestContext.BuildRID}/{Binaries.GetSharedLibraryFileNameForCurrentPlatform("native")}";
```

Comment:
I see `BuildRID` here, but `BUILD_RID` in the above MSBuild scripts. Are they the same thing? There is also `OutputRID`, which seems to map to some MSBuild script concept. Does the `TestContext.BuildRID` document how they are related?

Reply:
Nope. I think what you've done is improved this.